src/geolocation/component.js

```diff
@@ -230,10 +250,21 @@ function Controller(
     }
   );
 
-  const view = map.getView();
+  const view = this.map.getView();
 
   listen(view, 'change:center', this.handleViewChange_, this);
   listen(view, 'change:resolution', this.handleViewChange_, this);
+
+  if (this.options.atloadingtime && typeof this.loading != 'undefined') {
```

Comment:
`'undefined'` shouldn't be `undefined` (without quote)?

Reply:
Do you mean
```
if (this.options.atloadingtime && this.loading !== undefined) {
```
?

Reply:
+1 Yes :-)